### PR TITLE
KB migration: Change New Relic account owner

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) adds a new section about changing the New Relic account owner on a Magento Commerce (Cloud) project.

Specifically, it moves content from the [existing Magento Knowledgebase (KB) article](https://support.magento.com/hc/en-us/articles/360006643192-Change-account-owner-for-New-Relic-on-Cloud) to the DevDocs site.

Similar to #5070. 

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/cloud/project/new-relic.html
- https://devdocs.magento.com/guides/v2.2/cloud/project/new-relic.html

whatsnew
Added a new section about [changing the New Relic account owner](https://devdocs.magento.com/guides/v2.3/cloud/project/new-relic.html#change-account-owner) on a Magento Commerce (Cloud) project.

### CODE REVIEW

File: guides/v2.2/cloud/project/new-relic.md [hunk 1]
```diff
@@ -149,3 +149,13 @@ You can use the high-level data captured by New Relic APM to see what is happeni
 -  The New Relic worst-case requests are not averages, but edge cases; whereas, Blackfire provides more of the average worst-case requests.
 -  Blackfire is not detailed in determining and displaying queries giving you trouble, but you can use New Relic to find those.
 -  Background processes, like Cron job, can cause issues. These background processes are not monitored by New Relic, but you can watch them using Blackfire.
+
+## Change account owner
+
+See [New Relic's documentation][1].
+
+Make sure the user that you want to assign the Owner role to is listed on your {{site.data.var.ece}} account. See [Add a new user][2]. If a user needs to be added, but no existing Owner or Admin on the account can help, any Magento user who has access to New Relic's [Magento Partnership Owner Account][3] can add users on the customer's behalf.
```
Comment (hguthrie) [inline guides/v2.2/cloud/project/new-relic.md]:
```suggestion
Before assigning the Owner role to a user, verify the user exists on your {{site.data.var.ece}} account. See [Add a new user][2]. If you need to add the user, but an existing account Owner or Admin cannot help, any Magento user with access to New Relic's [Magento Partnership Owner Account][3] can add users on the behalf of the customer.
```

File: guides/v2.2/cloud/project/new-relic.md [hunk 2]
```diff
@@ -149,3 +149,13 @@ You can use the high-level data captured by New Relic APM to see what is happeni
 -  The New Relic worst-case requests are not averages, but edge cases; whereas, Blackfire provides more of the average worst-case requests.
 -  Blackfire is not detailed in determining and displaying queries giving you trouble, but you can use New Relic to find those.
 -  Background processes, like Cron job, can cause issues. These background processes are not monitored by New Relic, but you can watch them using Blackfire.
+
+## Change account owner
+
+See [New Relic's documentation][1].
```
Comment (hguthrie) [inline guides/v2.2/cloud/project/new-relic.md]: Why? Seems early for this.

Comment (jeff-matthews) [reply guides/v2.2/cloud/project/new-relic.md]: The idea is to immediately direct readers New Relic's docs to learn how to do this. I didn't feel like a long explanation was necessary.

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**